test_autograph.sh

```diff
@@ -0,0 +1,90 @@
+#!/usr/bin/env bash
+
```

Comment:
Nit: `set -e` in order to stop the script if the download step fails, for instance.